User-Help/GlobalTrustLevels.md

```diff
@@ -0,0 +1,138 @@
+
+### Flags
+
+Flags send a user contribution to a moderation queue for review. Use these to report material inappropriate for a particular site, or anything that breaks the Code of Conduct or Terms of Service. 
+
+### Upvotes
+
+Upvotes help a post or answer gain visibility and act as an indicator of how the community values the contribution. They are also used in combination with downvotes to determine if it is well-received. 
+
+### Suggest edits
+
+Sometimes a user's contribution could use a little help. This could be due to grammar and formatting issues, or maybe the original poster has mentioned important information in a comment and needs to add it to the original post. Suggesting edits helps improve the quality of a site's content.
+
+### Suggest duplicates
+
+You may not have been the first person to ask this question! Duplicates aren't necessarily a bad thing, but for people looking for information it's helpful to have links to any previous posts on the same topic. 
+
+### Downvotes
+
+Downvotes are the negative analogue to upvotes. They are used to reduce the visibility of low quality contributions and are used in the calculation of whether or not a particular contribution is well-received. 
+
+### Vote-to-lock
+
+Some contributions need serious help. They might be off-topic, or unclear without additional information from the original poster. In these cases the best thing to do is often to lock the post so it's no longer able to accept responses until the original poster updates his work. All lock votes are based on certain categories, each associated with a linked community Meta post. A certain threshold of lock votes vs. no-lock votes must be met before a post is locked, and editing the post to adhere to site guidelines will allow it to be unlocked. 
+
+### Direct edits
+
+Sometimes a user isn't responsive or their post needs some help or it will be closed. In this case it's possible for the higher trust level users to make changes directly instead of just suggesting. 
+ 
+### Review flags and suggested edits
+
+Flags and suggested edits go into a moderation queue for approval. Higher trust level users can review these and take action if necessary. 
+
+### Create tags
+
+Creating tags is a difficult task, as they need to be generic enough to be commonly chosen, while also not duplicating other existing tags. Currently this functionality is limited to users with enough experience to have a good idea of what to look for in a tag. 
+
+### Protect posts
+
+If a post is controversial or has attracted low quality answers, sometimes the moderation team will restrict answering to users with a higher trust level.
+
+### Temporarily lock posts
+
+Similar to post protection, sometimes the moderation team needs to lock a post to keep discussion from getting out of hand, or to give the original poster a chance to add more information. 
```

Comment:
We've currently identified two types of locks: edits (to prevent edit wars) and comments (to prevent threads from going further off the rails or because things are exploding).  Locks don't affect whether a question can be answered.  Locks don't prevent voting (unlike on SE).